_docs/best-practices/best-practices.md

```diff
@@ -178,49 +132,24 @@ s:contributor:
     s:email: dyuen@not-oicr.on.ca
     s:name: Denis Yuen
 
-
 s:citation: https://figshare.com/articles/Common_Workflow_Language_draft_3/3115156/2
 s:codeRepository: https://github.com/common-workflow-language/common-workflow-language
 s:dateCreated: "2016-12-13"
 s:license: https://www.apache.org/licenses/LICENSE-2.0
-```
 
-For WDL descriptors, see the [WDL documentation](https://software.broadinstitute.org/wdl/documentation/spec#metadata-section) for how to define metadata.  
-
-Additionally, this following example includes author, email, and description metadata:
+s:keywords: http://edamontology.org/topic_0091 , http://edamontology.org/topic_0622
+s:programmingLanguage: C
 ```
-task runtime_meta {
-  String memory_mb
-  String sample_id
-  String param
 
-  command {
-    java -Xmx${memory_mb}M -jar task.jar -id ${sample_id} -param ${param} -out ${sample_id}.out
-  }
-  output {
-    File results = "${sample_id}.out"
-  }
-  runtime {
-    docker: "broadinstitute/baseimg"
-  }
-  parameter_meta {
-    memory_mb: "Amount of memory to allocate to the JVM"
-    param: "Some arbitrary parameter"
-    sample_id: "The ID of the sample in format foo_bar_baz"
-  }
-  meta {
-    author: "Denis Yuen"
-    email: "dyuen@not-oicr.on.ca"
-    description: "An example tool demonstrating metadata. Note that this is an example and the metadata is not necessarily consistent."
-  }
-}
-```
+## Input/Output Fields
 
-### Extended Example
+Tools and workflows can take *File* types as input and produce them as output. We recommend indicating the format for *File* types. This helps document for others how to use your tool while allowing you to do some simple type-checking when creating parameter files.
 
-For those that are highly motivated, it is also possible to annotate your tool with a much larger amount of metadata. This example includes EDAM ontology tags as keywords (allowing the grouping of related tools), hints at hardware requirements in order to use the tool, and a few more metadata fields.
+For file formats, we recommend referencing existing ontologies (like EDAM in our example), reference a local ontology for your institution, or do not add a file format initially for quick development before sharing your tool with others.
```

Comment:
I think there is a verb mismatch here, as you start off with `recommend referencing existing...`, but then go to `reference a`, and `do not add...`. I think they need to be in sync.

Something like, although I'm not wild about this:

`For file formats, we recommend referencing either existing ontologies (like EDAM in our example), or a local ontology for your institution. Or do not add a file format initially for quick development before sharing your tool with others.`